### PR TITLE
Feat/redesign modals

### PR DESCRIPTION
**Pull request by seibei-iguchi**
### Before
Screenshot 2024-09-24 at 10 50 40
Screenshot 2024-09-24 at 10 51 02
Screenshot 2024-09-24 at 10 51 07
Screenshot 2024-09-24 at 10 51 14

### After
Screenshot 2024-09-24 at 10 51 44
Screenshot 2024-09-24 at 10 43 50
Screenshot 2024-09-24 at 10 44 03
Screenshot 2024-09-24 at 10 44 09

### CODE REVIEW

**Comment by peter-sanderson** [comment]
I have unassigned myself, sorry but I have no time currently. I think @jvaclavik is enough :)

File: packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep3.tsx [hunk 1]
```diff
@@ -18,21 +18,12 @@ export const PassphraseWalletConfirmationStep3 = ({
...
-                    id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP3_WARNING"
-                    values={{
-                        bold: chunks => (
-                            <Text variant="warning" typographyStyle="callout">
-                                {chunks}
-                            </Text>
-                        ),
-                    }}
-                />
+                <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP3_WARNING" />
```
Comment (jvaclavik) [inline packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep3.tsx:26]: Shouldn't we keep it? 🤔

**Review by jvaclavik — APPROVED** [review]
I really like this refactoring!

File: packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep3.tsx (hunk 1 above)
Comment (seibei-iguchi) [reply packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep3.tsx:26]: No bold in the translation string

**Comment by STew790** [comment]
**QA OK**
LGTM
image
image
image
image

**Info:**
 - Suite version: desktop 24.10.1 (3017e5864a176715b1023945397ef49e8aa24d5e)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.10.1 Chrome/126.0.6478.234 Electron/31.6.0 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T3T1 2.8.3 regular (revision 7f373ae71eca855dd41b4a0fdcc7cadaa13a8281)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**